src/live.rs

```diff
@@ -43,6 +42,10 @@ pub fn iso_remove(config: &IsoIgnitionRemoveConfig) -> Result<()> {
 }
 
 pub fn iso_ignition_embed(config: &IsoIgnitionEmbedConfig) -> Result<()> {
+    if config.output.as_deref() == Some("-")
```

Comment:
I suspect this shouldn't be magic but wasn't certain the correct way to define it...maybe, `const USE_STDOUT: str = "-"`?